core/src/avm2/verify.rs

```diff
@@ -545,6 +545,17 @@ impl<'gc> Locals<'gc> {
         self.0[index] = ValueType::Class(class);
     }
 
+    fn set_class(&mut self, index: usize, class: GcCell<'gc, Class<'gc>>) {
+        // FIXME: Getting the ClassObject this way should be unnecessary
+        // after the ClassObject refactor
+        self.0[index] = class
+            .read()
+            .class_objects()
+            .get(0)
```

Comment:
I'm not sure what the behavior of Flash is when multiple `ClassObjects` have been created. To be safe, we should do nothing in this method when that occurs (and also log a warning).

Reply:
The `VTable`s should be the same, though, no?

Reply:
I'm not certain - as long as we're using `ClassObject`, I would prefer to avoid making assumptions about when distinct `ClassObjects` can be freely intermixed